### DISCUSSION TITLE
Delete an unmerged pull request

### DISCUSSION THREAD

**Discussion by kazitek** [community/community#23751] [New to GitHub]
<p>Hey Github, how do I delete an accidental PR that has not yet been merged by the destination repository?. Accidentally created a pull request its not merged but am unable to completely delete it.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/kazitek">@kazitek</a>! Welcome to the community! Pull requests can’t be deleted through the UI at the moment and our private support team deletes pull requests <code>only</code> when they contain sensitive information like passwords or other credentials.</p>
<p>In cases like this, it’s fine to just close the pull request and leave a comment explaining why it was closed. I hope this helps!</p>

**Reply by justinvirtualitics** [reply]
How can this be done by administrators on a Github Enterprise Server?

**Reply by aiadam36** [reply]
How can I do so?

**Comment by kazitek** [comment]
<p>Hey, <a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a>, this particular PR doesn’t have any sensitive information, I just created it from the wrong linked account.<br>
I will use the option of leaving a comment, thanks. :+1:</p>

**Comment by BugOfBugs** [comment]
Delete <code>Repository</code>. Create new same.
>@AndreaGriffiths11 deletes pull requests only

Why?

**Comment by truongtamthanh2004** [comment]
you can close pull request and leave a comment. I hope it helps.
